infrastructure/analytics/plausible-conf.env.example

```diff
@@ -0,0 +1,5 @@
+ADMIN_USER_EMAIL=email@example.com
```

Comment:
because of the way our hosting work, would you be able to move these .env variables into the main .env at the root level please 👍

Reply:
This is the configuration if you wanted to stand up your own server - for your own hosting I guess you would want to sort out your own hosting instance?  The problem with Docker (which is what I've used to stand this infrastructure up) is that it can't access files outside the context folder